docs/topics/maven-arguments.adoc

```diff
@@ -14,11 +14,11 @@ The following is a detailed description of the available {ProductShortName} {Mav
 |explodedApps |Flag to indicate that the provided input directory contains source files for a single application. See the xref:input_file_type_arguments[Input File Argument Tables] for details.
 |exportCSV |Flag to export the report data to a CSV file on your local file system. {ProductShortName} creates the file in the directory specified by the `outputDirectory` argument. The CSV file can be imported into a spreadsheet program for data manipulation and analysis. For details, see xref:export_csv[Export the Report in CSV Format].
 |includeTags |A list of tags to use. When specified, only rules with these tags will be processed. 
-|inputDirectory |Specify the path to the directory containing the applications to be analyzed. This argument defaults to the project's source directory. See xref:input_argument[Specify the Input Directory] for more information.
+|inputDirectory |Specify the path to the directory containing the applications to be analyzed. This argument defaults to `{project.basedir}/src/main/`. See xref:input_argument[Specify the Input Directory] for more information.
```

Comment:
I think this section needs documentation for the new "windupHome" parameter as well.

Reply:
This parameter has now been included.